src/Horarium.IntegrationTest/TestParallelsWorkTwoManagers.cs

```diff
@@ -18,14 +16,14 @@ public class TestParallelsWorkTwoManagers : IntegrationTestBase
         public TestParallelsWorkTwoManagers()
         {
             var provider = new MongoClientProvider(ConnectionMongo);
-            var collection = provider.GetCollection<JobDb>();
+            var collection = provider.GetCollection<JobMongoModel>();
```

Comment:
Why integration test is stuck to Mongo DB provider?
Maybe we should rename the tests to reflect that these tests are for MongoDB?
Will be there same integration tests for other DB providers?

Reply:
I should exist this test for all providers, I will add TestCase

Reply:
done